public/translations/en.json

```diff
@@ -11,6 +11,17 @@
     "or_choose": "Or choose",
     "no_image_available": "No image available",
     "no_items_found": "There are no items found.",
+    "collection_url_error": "Collection Error",
+    "manifest_url_error": "Manifest Error",
+    "item_url_error": "Item Error",
+    "url_error": "The URL {{url}} is provided wrong",
+    "data_error": "Wrong data",
+    "data_provided_error": "The data are provided wrong",
+    "selector_error": "Selector Error",
+    "referenced_selector_provided_error": "The selector of referenced element is provided wrong",
+    "referenced_element_not_found": "The referenced element could not be found since its selector was provided wrong",
+    "content_type_error": "Content Type error",
+    "referenced_content_type_error": "The content type of $t(item) where ref.element lies is provided wrong",
```

Comment:
This is not a good error message for the users. It is better not to relate to code internals but explain what happened in a **short** and precise manner.